### PR TITLE
Ignore annotated coverage files

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ pip-wheel-metadata
 .vscode
 Pipfile
 Pipfile.lock
+*,cover
```
Comment (dbutenhof) [inline .gitignore:23]: Do these show up anywhere, or should they be anchored to the base directory as `/*,cover`??

**Review by dbutenhof — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (portante) [reply .gitignore:23]: They show up where the `.py` is when you ask for them using `annotate` command.

Comment (dbutenhof) [reply .gitignore:23]: Interesting; I always just pull up the annotated html version in the browser, but that could be convenient. I'll have to try it.

Comment (dbutenhof) [reply .gitignore:23]: Ah; OK, that's nice to know, but I like the html version better. (Probably just the color, though!)

image

vs 

image

**Review by ndokos — APPROVED** [review]

**Pull request merged**